ports/openxr-loader/portfile.cmake

```
@@ -44,11 +51,29 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
+set(ENV{OPENXR_REPO} ${SDK_SOURCE_PATH})
+
+vcpkg_execute_required_process(
+    COMMAND ${PYTHON3} ${HPP_SOURCE_PATH}/scripts/hpp_genxr.py -registry ${SDK_SOURCE_PATH}/specification/registry/xr.xml -o ${CURRENT_PACKAGES_DIR}/include/openxr openxr.hpp
+    WORKING_DIRECTORY ${HPP_SOURCE_PATH}
+    LOGFILE openxrhpp
+)
+
+vcpkg_apply_patches(
+    SOURCE_PATH ${CURRENT_PACKAGES_DIR}/include/openxr
+    PATCHES
+        001-fix-array-decl.patch
+)
+
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/${PORT})
+else(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake TARGET_PATH share/${PORT})
+endif()
```

```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/OpenXR)
else(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/openxr TARGET_PATH share/OpenXR)
endif()
```
Since the CMake configs and targets files are like this:
```
OpenXRConfigVersion.cmake
OpenXRConfig.cmake
OpenXRTargets.cmake
OpenXRTargets-release.cmake
OpenXRConfigVersion.cmake
OpenXRConfig.cmake
OpenXRTargets.cmake
OpenXRTargets-debug.cmake
```
In this case, the usage info will be updated as the follows:

```
find_package(OpenXR CONFIG REQUIRED)
target_link_libraries(main PRIVATE OpenXR::headers OpenXR::openxr_loader OpenXR::openxr-all-supported)
```